app/src/main/res/layout/activity_settings.xml

```diff
@@ -190,6 +190,7 @@
                     android:text="@string/show_clipboard_content" />
 
             </RelativeLayout>
+
             <RelativeLayout
                 android:id="@+id/settings_show_numbers_row_holder"
```

Comment:
I tried to follow the style which was here, but I was looking only at the item above. I should change it's name too, if that should be without the "row"